### PR TITLE
Lra 525 lsa ride share edit styles for program pages

### PR DESCRIPTION
**Pull request by britaumich**
All design changes were made according to Maria's comments in this ticket [LRA-525](https://umlsait.atlassian.net/browse/LRA-525).
Another pair of eyes would be helpful to check if I missed something.

### CODE REVIEW

**Review by rsmoke — CHANGES_REQUESTED** [review]
I am going back and fixing a number of apps to include Bear River, MMSS, NELP. I did not take the time to setup the system in a way that I was"telling"what I want at a particular point. 
Fixing these logic bombs now will prevent technical debt in the too near future.

Getting the conditionals out of the code and coming from a perspective of the tell don't ask will keep up sane.
http://rubyblog.pro/2016/09/tell-dont-ask-principle

File: app/models/program.rb [hunk 1]
```diff
@@ -57,7 +59,7 @@ def display_name
     if self.not_course
       "This program is not a course"
     else
-      "#{self.subject} #{self.catalog_number} - #{self.class_section}"
+      "#{self.subject} #{self.catalog_number} - #{self.class_section} - #{self.term.name}"
```
[the remark this answers is not in the document]
Comment (britaumich) [reply app/models/program.rb:83]: @rsmoke  Here we create a program's display_name. The name contains some fields if they present or a phrase "This program is not a course". I don't know how to change that without using conditions.
```
def display_name
    if self.not_course
      "This program is not a course - #{self.term.name}"
    else
      "#{self.subject} #{self.catalog_number} - #{self.class_section} - #{self.term.name}"
    end
end
```

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -29,13 +29,13 @@ def program_config_data(program)
     if program.pictures_required_start || program.pictures_required_end
       config_data = 'The program requires to upload pictures to the vehicle reports '
       if program.pictures_required_start
```
[the remark this answers is not in the document]
Comment (britaumich) [reply app/helpers/application_helper.rb]: @rsmoke I moved the method of creating the text with additional options to the Program model. But I don't see how to remove the conditions. The text is created depending on the field values' existence in a record. There are three boolean fields that determine the final text.

File: app/controllers/programs_controller.rb [hunk 3]
```diff
@@ -11,10 +11,8 @@ def index
     @terms = Term.all
     if params[:term_id].present?
       @programs = Program.where(term_id: params[:term_id])
-      @term_id = params[:term_id]
     else
-      @programs = Program.all
-      @term_id = nil
+      @programs = Program.current_term
```
[the remark this answers is not in the document]
Comment (britaumich) [reply app/controllers/programs_controller.rb]: @rsmoke I created a scope in the Program model to get programs without checking term_id existence in the controller.

**Pull request merged**